pkg/util/common_test.go

```diff
@@ -111,15 +139,26 @@ func TestStringerJoin(t *testing.T) {
 }
 
 func TestKeys(t *testing.T) {
-	mcs := map[string]sigMultiCluster{
-		kubefed.Name: kubefed,
-		karmada.Name: karmada,
-	}
-	got := Keys(mcs)
-	expect := []string{kubefed.Name, karmada.Name}
-	sort.Strings(got)
-	sort.Strings(expect)
-	if !reflect.DeepEqual(got, expect) {
-		t.Errorf("got = %v, want %v", got, expect)
-	}
+	t.Run("keys exist", func(t *testing.T) {
+		mcs := map[string]sigMultiCluster{
+			kubefed.Name: kubefed,
+			karmada.Name: karmada,
+		}
+		got := Keys(mcs)
+		expect := []string{kubefed.Name, karmada.Name}
+		sort.Strings(got)
+		sort.Strings(expect)
+		if !reflect.DeepEqual(got, expect) {
+			t.Errorf("got = %v, want %v", got, expect)
+		}
+	})
+	t.Run("empty keys", func(t *testing.T) {
+		var mcs map[string]sigMultiCluster
+		got := Keys(mcs)
+		var expect []string
+		sort.Strings(got)
+		if !reflect.DeepEqual(got, expect) {
+			t.Errorf("got = %v, want %v", got, expect)
+		}
+	})
```

Comment:
How about rewriting this test as tabel drive as well? Just like your rewrite above.

Reply:
I thought the same. Now I'm doing it...